pkg/apis/migration/v1alpha1/migcluster_types.go

```diff
@@ -218,6 +220,23 @@ func (m *MigCluster) GetClusterSubdomain(c k8sclient.Client) (string, error) {
 	return clusterSubdomain, nil
 }
 
+// GetRegistryImage gets a MigCluster specific registry image from ConfigMap
+func (m *MigCluster) GetOperatorVersion(c k8sclient.Client) (string, error) {
+	client, err := m.GetClient(c)
+	if err != nil {
+		return "", err
```

Comment:
Replace err returns with 

```
return "", liberr.Wrap(err)
```

This provides stack trace from full call stack when error returned down the chain is finally displayed